src/mono/System.Private.CoreLib/src/System/Diagnostics/StackFrame.Mono.cs

```diff
@@ -51,5 +51,11 @@ private void BuildStackFrame(int skipFrames, bool needFileInfo)
         private static extern bool get_frame_info(int skipFrames, bool needFileInfo,
             out MethodBase method, out int ilOffset, out int nativeOffset, out string file, out int line, out int column);
 
+        /// <summary>
+        /// Returns the method info instance for the managed code IP address.
+        /// </summary>
+        /// <param name="ip">code address</param>
+        /// <returns>MethodBase instance for the method or null if IP not found</returns>
+        internal static MethodBase? GetMethodFromNativeIP(IntPtr ip) => throw new NotSupportedException();
```

Comment:
It would be better to fix this build break by moving the ILLinker directive to CoreCLR specific file: src\coreclr\System.Private.CoreLib\src\ILLink\ILLink.Descriptors.Shared.xml . Nothing in Mono is affected that way by the servicing fix.